gallery/src/components/NxStatefulTabs/NxStatefulTabsPage.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow
+} from '@sonatype/react-shared-components';
+
+import NxStatefulTabsSimpleExample from './NxStatefulTabsSimpleExample';
+
+const NxStatefulTabsSimpleCode = require('!!raw-loader!./NxStatefulTabsSimpleExample').default;
+
+export default function NxTablePage() {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <section className="nx-tile--subsection">
+          <p className="nx-p">
+            This component simply wraps the <code className="nx-code">NxTabs</code> component
+            to track the currently selected tab.
+          </p>
+
+          <NxTable className="nx-table--gallery-props">
+            <NxTableHead>
+              <NxTableRow>
+                <NxTableCell>Prop</NxTableCell>
+                <NxTableCell>Type</NxTableCell>
```

Comment:
These tables usually have a column for whether or not the prop is required.

Reply:
140a510